### PR TITLE
Update readme

### PR DESCRIPTION
**Pull request by marcelveldt**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,21 +1,161 @@
...
+This project implements a Matter Controller Server over WebSockets using the [official CHIP SDK](https://github.com/project-chip/connectedhomeip) as a base and provides both a server and client implementation.
 
-This repository is for development only. For enabling Matter support within Home Assistant, please refer to the Home Assistant documentation.
+The goal of this project is primary to have Matter support in Home Assistant but its universal approach makes it suitable to be used in other projects too.
 
-:warning: This project is under development and considered BETA !
+This repository is for development only (so not for enduser support). For enabling Matter support within Home Assistant, please refer to the [Home Assistant documentation](https://www.home-assistant.io/integrations/matter/).
+
+NOTE: Both Matter and this implementation are in early (v1) state and probably features are missing or could be improved. See our [project board](https://github.com/orgs/home-assistant-libs/projects/1) for status updates and maybe something you'd like to help out with development and/or testing.
```
Comment (MartinHjelmare) [inline README.md:9]:
```suggestion
NOTE: Both Matter and this implementation are in early (v1) state and features are probably missing or could be improved. See our [project board](https://github.com/orgs/home-assistant-libs/projects/1) for status updates and maybe something you'd like to help out with, development and/or testing.
```

File: README.md [hunk 2]
```diff
@@ -1,21 +1,161 @@
...
+
+### Test devices
+
+Now that the Matter Specification is offically released in its 1.0 version, devices will be available in stores from 2023 that actually have Matter support or least manufacturers run a beta program which you can join to run Matter firmware on your device. Please refer to the documentation of your device if its already Matter enabled out of the box or you need to enable some special firmware(mode).
+
+Besides that it is possible to run Matter firmware on common microcontrollers such as the ESP32 and there is even a whole device emulator available which runs on regular desktop OS. To make things easier we've prepared a [special page](https://nabucasa.github.io/matter-example-apps) where you can quickly try our running the Matter example apps on ESP32.
```
Comment (MartinHjelmare) [inline README.md:44]:
```suggestion
Besides that it is possible to run Matter firmware on common microcontrollers such as the ESP32 and there is even a whole device emulator available which runs on a regular desktop OS. To make things easier we've prepared a [special page](https://nabucasa.github.io/matter-example-apps) where you can quickly try out running the Matter example apps on ESP32.
```

File: README.md (hunk 1 above)
Comment (MartinHjelmare) [reply README.md:9]: Maybe link to the development section since we write the same there?

Comment (marcelveldt) [reply README.md:9]: something like this? (I've edited your text)

Comment (MartinHjelmare) [reply README.md:9]: Great!

**Review by MartinHjelmare — APPROVED** [review]

**Pull request merged**